snuba/clickhouse/query_dsl/accessors.py

```diff
@@ -134,7 +134,8 @@ def get_time_range_expressions(
 
 
 def get_time_range(
-    query: ProcessableQuery[Table], timestamp_field: str
+    query: Union[ProcessableQuery[Table], ProcessableQuery[Entity]],
```

Comment:
Do you need this change anymore? I think we ended up moving the storage selection to later in the pipeline where we already have a table.

Reply:
Yes we still need it because this instance of `Query` -> https://github.com/getsentry/snuba/blob/6cd74121601df0f4dbef6590a3b04899847a7559/snuba/datasets/entities/sessions.py#L212 
is an instance of `class Query(AbstractQuery[Entity]):`

Reply:
Ah, right